repo-index.yml

```
@@ -339,6 +339,40 @@ plugins:
   name: cf-icd-plugin
   updated: "2017-04-04T00:00:00Z"
   version: 0.0.11
+- authors:
+  - contact: daria.anton@sap.com
+    homepage: https://github.com/Dariquest
+    name: Daria Anton
+  - contact: alexander.lais@sap.com
+    homepage: https://github.com/peanball
+    name: Alexander Lais
+  - contact: maximilian.moehl@sap.com
+    homepage: https://github.com/maxmoehl
+    name: Maximilian Möhl
```

```suggestion
  - contact: cf@moehl.eu
    homepage: https://github.com/maxmoehl
    name: Maximilian Moehl
```

Just some minor adjustments :)

---

done